src/frontends/pytorch/src/op/sort.cpp

```diff
@@ -0,0 +1,52 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <climits>
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset10.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+#define mark(...) context.mark_node(__VA_ARGS__)
+#define shared    std::make_shared
+
+using namespace opset10;
+
+OutputVector translate_sort(NodeContext& context) {
+    const auto input_tensor = context.get_input(0);
+    bool stable;
+    int64_t dim;
+    bool descending;
+
+    if (context.get_input_size() == 4) {
+        stable = context.const_input<bool>(1);
+        FRONT_END_OP_CONVERSION_CHECK(stable == false, "Stable sorting in aten::sort is not yet supported.");
+        dim = context.const_input<int64_t>(2);
+        descending = context.const_input<bool>(3);
+    } else {
+        dim = context.const_input<int64_t>(1);
+        descending = context.const_input<bool>(2);
+    }
+    auto mode = descending ? ov::op::TopKMode::MAX : ov::op::TopKMode::MIN;
```

Comment:
may be the following helps to make it shorter

```
using namespace ov::op;
```

Reply:
Good idea, applied!